pinax/stripe/tests/test_actions.py

```diff
@@ -279,6 +280,7 @@ def setUp(self):
             interval_count=1,
             name="Pro"
         )
+        self.account = Account.objects.create(stripe_id="acc_XXX")
```

Comment:
Would move it to `test_purge_connected` - no need to create this for every test.